backend/data/src/main/kotlin/io/tolgee/service/NotificationService.kt

```diff
@@ -0,0 +1,21 @@
+package io.tolgee.service
+
+import io.tolgee.repository.NotificationRepository
+import org.springframework.stereotype.Service
+
+@Service
+class NotificationService(
+  private val notificationRepository: NotificationRepository,
+) {
+  fun getNotifications(userId: Long): List<NotificationModel> {
+    return notificationRepository.fetchNotificationsByUserId(userId)
+      .map { NotificationModel(it.id, it.linkedTask?.project?.id, it.linkedTask?.number, it.linkedTask?.name) }
+  }
+}
+
+data class NotificationModel(
+  val id: Long,
+  val linkedProjectId: Long?,
```

Comment:
I would name it just "projectId". So we keep the "linked" prefix only for the "notification event data"